main/res/values/changelog_master.xml

```diff
@@ -4,7 +4,12 @@
   <string name="changelog_master" translatable="false"><Data><![CDATA[<div>
 <h1>Nightly build:</h1>
 
-*** TODO: Need some introductionary text about the SAF change ***
+<h2>
+We had to do extensive changes to the way c:geo gets access to your file system (e.g. for storing data, reading offline maps) to be prepared for requirements on new Android versions.
```

Comment:
Reformulation suggestion:
Starting with Android 11, Google enforces usage of <a href="https://developer.android.com/about/versions/11/privacy/storage">Scoped Storage</a> to access public file system on mobile devices. 
C:geo uses this file system to store and access data like e.g. offline maps, GPX files, backups and other data.  Major changes were done to comply to the new access system.  
After upgrading to this c:geo version some user action is necessary to allow c:geo accessing necessary public folders. A wizard will guide you through the needed changes.  Please follow these steps in the wizard carefully!

Reply:
To long and to much technology terms IMHO

Reply:
Ok, another try:
To comply to new Android versions, changes where necessary to the way c:geo accesses the file system (e.g. for storing data, reading offline maps). As a consequence you will need to grant folder access permissions to c:geo. 
A wizard will guide you through the needed changes. Please follow the steps in the wizard carefully!

Reply:
This wording would not be that personal. While the app content should be focused to the user side, a changelog is a summary of what the team has done for c:geo IMHO. Therefore I like "we" and @Lineflyer's version in total.

Reply:
Go for it

Reply:
I guess that >80% of people will anyhow skip the screen without reading...I will also place this sentence into the PlayStore changelog...but still. Don't expect many people reading it.

So lets merge and thanks for your comments.